exe/bundle_report

```diff
@@ -1,78 +1,6 @@
 #!/usr/bin/env ruby
-#
-# Print a report on our Gemfile
-# Why not just use `bundle outdated`? It doesn't give us the information we care about (and it fails).
-#
-at_exit do
```

Comment:
Was there an specific reason for using this `at_exit` block?
I removed it from `def run` in the CLI because it was making tests execution fail for it